File: OOP Lab Final/lab6.java
import java.util.Scanner;

class StringAdd {
    public String s1, s2;

    public void setString() {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter string s1");
        s1 = sc.nextLine();
        System.out.println("Enter string s2");
        s2 = sc.nextLine();
        String s3 = s1 + s2;
        System.out.println("Add two strings = " + s3);
        sc.close();
    }

}

public class lab6 {
    public static void main(String[] args) {
        StringAdd obj = new StringAdd();
        obj.setString();
    }

}

File: OOP_C++_JAVA/lab6.cpp
#include <bits/stdc++.h>
using namespace std;
class string_add
{
public:
    string s1, s2;
    void set_string()
    {
        cout << "Enter string s1" << endl;
        cin >> s1;
        cout << "Enter string s2" << endl;
        cin >> s2;
        string s3 = s1 + s2;
        cout << "Add two strings = " << s3 << endl;
    }
};
int main()
{
    string_add obj;
    obj.set_string();
    return 0;
}
